sleap/gui/dialogs/formbuilder.py

```diff
@@ -424,6 +446,7 @@ def build_form(self, items_to_create: List[Dict[Text, Any]]):
             return
         for item in items_to_create:
             self.add_item(item)
+       
```

Comment:
Running `black` on this file will format this for us - either through vscode's black extension + `formatOnSave` setting or through the CLI:

```bash
black  sleap/gui/dialogs/formbuilder.py
```
.
```suggestion
```